src/server/db_slice.h

```diff
@@ -331,7 +331,7 @@ class DbSlice {
   constexpr static DbIndex kDbAll = 0xFFFF;
 
   // Flushes db_ind or all databases if kDbAll is passed
-  void FlushDb(DbIndex db_ind);
+  util::fb2::JoinHandle FlushDb(DbIndex db_ind);
```

Comment:
why could not you return a fiber? why do you need a handle?

Reply:
Ahhhh... More elegant interface because keeping the default behaviour doesn't require any action from callees. But I've just implemented it without and it's fine